FrontEnd/src/components/profileList/ProfileCard.jsx

```diff
@@ -0,0 +1,145 @@
+import css from "./ProfileCard.module.css";
+import { Badge, Typography } from "antd";
+import { useState } from "react";
+import { StarOutlined, StarFilled } from "@ant-design/icons";
+import { PropTypes } from "prop-types";
+import axios from "axios";
+import { useSWRConfig } from "swr";
+const { Paragraph } = Typography;
+
+// FIXME: will be changed once serializer will be changed to return display_name
+const regions = [
+  {
+    key: "Lviv region",
+    value: "Львівська область",
+  },
+  {
+    key: "Kyiv region",
+    value: "Київська область",
+  },
+];
+
+export default function ProfileCard(props) {
+  const isAuthorized = props.isAuthorized;
+  const data = props.data;
```

Comment:
Do we really need this reassignment?

Reply:
Thanks, replaced reassignment with descructuring of props :)